### PR TITLE
feat: add Storybook

### PR DESCRIPTION
**Pull request by riccardo-forina**
Screen Shot 2019-09-24 at 6 05 06 PM

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const appConfig = require('../webpack.common');
+
+module.exports = ({ config }) => {  
+  config.module.rules.push(...appConfig.module.rules);
```
Comment (seanforyou23) [inline .storybook/webpack.config.js]: Do we need to register `react-docgen-typescript-loader` here?

File: package.json [hunk 2]
```diff
@@ -42,6 +52,7 @@
     "raw-loader": "^3.1.0",
     "react": "^16.8.4",
     "react-axe": "^3.0.2",
+    "react-docgen-typescript-loader": "^3.2.1",
```
Comment (seanforyou23) [inline package.json]: Maybe I'm overlooking it, but where do we benefit from this package? Is it supposed to generate prop tables below the storybook entries?

File: package.json [hunk 3]
```diff
@@ -15,18 +15,28 @@
...
     "@types/enzyme-adapter-react-16": "^1.0.5",
     "@types/jest": "^24.0.17",
     "@types/node": "^12.7.1",
     "@types/react": "^16.8.25",
     "@types/react-dom": "^16.8.5",
     "@types/react-router-dom": "^4.3.4",
+    "@types/storybook__react": "^4.0.2",
     "@types/victory": "^31.0.14",
     "@types/webpack": "^4.32.1",
+    "babel-loader": "^8.0.6",
```
[the remark this answers is not in the document]
Comment (riccardo-forina) [reply package.json]: Nice catch! I think this comes from a yarn add copy-pasted without reading. It's not actually used, so all babel related stuff can go away.

File: .storybook/webpack.config.js (hunk 1 above)
Comment (riccardo-forina) [reply .storybook/webpack.config.js]: Right, will add.

File: package.json (hunk 2 above)
Comment (riccardo-forina) [reply package.json]: Yes, I'll update stories to make use of it.

**Comment by riccardo-forina** [comment]
Thanks for the feedback! The warning bit was tricky, Storybook's own Webpack's config was providing some default CSS loader that wasn't compatible with the one needed for PatternFly.

**Pull request merged**